pkg/serialization/codec/jam/decode.go

```diff
@@ -50,6 +87,13 @@ func (br *byteReader) unmarshal(value reflect.Value) error {
 		return br.decodeFixedWidth(value, l)
 	case []byte:
 		return br.decodeBytes(value)
+	case []bool:
```

Comment:
I think we have to differentiate boolean slice from a bit sequence. I suggest creating another type for handling this case, maybe something like this: 
`type BitSequence []bool`